src/components/ReportsPage/AllScorecardsPage/AllScorecardsHeatmap.tsx

```diff
@@ -48,7 +51,20 @@ export const AllScorecardsHeatmap = ({
     );
   }, [scorecards]);
 
-  if (loading) {
+  const { value: entities, loading: loadingEntities } = useCortexApi(
+    api => api.getCatalogEntities(),
+    [],
+  );
+
+  const entitiesByTag: StringIndexable<HomepageEntity> = useMemo(
```

Comment:
OK last comment: I think we have this in enough places to just create a utility hook that all of these usages uses instead of copy/pasting it.